client/src/index.css

```
@@ -90,3 +90,41 @@
     @apply bg-yellow-500;
   }
 }
+
+
+@keyframes xiPulseActive {
```

Stylelint (17.6.0)

[error] Expected keyframe name "xiPulseActive" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)